canonical/greenhouse-automations
# Updated regions

## Done
- Regions are updated.

## QA
- Run `ts-node ./src/index.ts add-job -i`
- Select one of the jobs.
- Select one of the job posts.
- Verify "us-boston" region is displayed

Fixes #38
